perfkitbenchmarker/benchmark_sets.py

```diff
@@ -190,7 +191,7 @@ def GetBenchmarksFromFlags():
     else:
       benchmark_names.add(benchmark)
 
-  # Expand recursive sets
+# Expand recursive sets
```

Comment:
Ah, you're right. That's weird, and I think we should change that to tox - it doesn't look like the current implementation actually checks the installed version. To get around it, you can temporarily disable that hook, or you can install the flake8 and pep8 versions specified in the tox.ini file:
```
pip install pep8==1.5.7
pip install flake8==2.4.1
```
If you're not working in a virtualenv, you might need sudo for those commands.